Tetris Clone C++/TetrisPiece.cpp

```
@@ -91,6 +109,7 @@ bool TetrisPiece::rotateClockwise() {
         if (width == height) {
             bounds = width;
         } else {
+            // general error
```

maybe raise an exception